services/metadactyl-clj/src/metadactyl/service/apps/de/jobs.clj

```diff
@@ -38,9 +37,11 @@
   [job]
   (try+
    (jex/submit-job (pre-process-jex-submission job))
+   ; FIXME: Should we let this error ripple up? Not catch it?
```

Comment:
The intent of the catch here was just to log the fact that the job submission failed. If we can remove the catch here while still keeping it apparent that a job submission failed in the logs then I don't see any reason to catch the exception here. The one caveat is that this may not be called in the context of a service call initiated by the user. If the JEX job happens to be a step in a mixed pipeline then it could be in response to job polling activity or a callback.